src/main/java/uk/gov/hmcts/divorce/notification/CommonContent.java

```diff
@@ -114,18 +114,6 @@ public String getPartner(CaseData caseData, Applicant partner) {
         }
     }
 
-    public String getPartnersSolicitor(CaseData caseData, Applicant partner) {
```

Comment:
this seems like itll be useful, why remove it?

Reply:
because it doesn't do much, it's the same as getPartner, just adds " 'solicitor" basically. so we can just use the other method
also, it was redoing all the logic that the getPartner method was doing